src/Caching/Test/ClassesTests.cs

```diff
@@ -150,5 +150,47 @@ class B(A):
             // In JSON, class A should have 'class A doc' documentation while B should have none.
             Baseline.CompareToFile(BaselineFileName, json);
         }
+
```

Comment:
todo: need to remove theses now that they have their own file.


Also I'm worried about persistence and base class is namedtuple because classModel special cases that